qpdf/qtest/qpdf-json.test

```diff
@@ -356,18 +356,18 @@ $td->runtest("handle binary names",
 $td->runtest("check json",
              {$td->FILE => "a.json"},
              {$td->FILE => "weird-tokens.json"},
-             $td->NORMALIZE_NEWLINES);
+             $td->EXPECT_FAILURE);
```

Comment:
For future reference, if you want to illustrate a bug, this shuld be `$td->NORMALIZE_NEWLINES | $td->EXPECT_FAILURE` to reduce the likelihood of forgetting to add `$td->NORMALIZE_NEWLINES` when fixing the bug.